samples/AzureSDK/AzureSDKSensorCertificate/NFSensorAzureIoTHub/app.config

```diff
@@ -0,0 +1,47 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
we shouldn't have app.config in nano projects... Please remove

Reply:
Thanks Jose, I removed app.config files